nautilus_trader/adapters/interactive_brokers/client/client.py

```diff
@@ -162,14 +162,14 @@ def _start(self) -> None:
         message reader, and internal message queue processing tasks.
 
         """
-        self._log.info(f"Starting InteractiveBrokersClient ({self._client_id})...")
         if not self._loop.is_running():
             self._log.warning("Started when loop is not running.")
-            self._loop.run_until_complete(self._startup())
+            self._loop.run_until_complete(self._async_start())
         else:
-            self._create_task(self._startup())
+            self._create_task(self._async_start())
 
-    async def _startup(self):
+    async def _async_start(self):
```

Comment:
Do we have synchronous versions of the start and stop methods? otherwise the `_async` prefix is a little redundant, and usually a postfix when there is also a synch version.

Reply:
`InteractiveBrokersClient` inherits from `Component`, which implements the `start()`, `stop()`, `reset()` and `resume()` synchronous methods and also defines `_start()`, `_stop()`,  `_reset()` and `_resume` as abstract synchronous methods to be overridden in subclasses. Because `InteractiveBrokersClient` needs to execute async methods, we have to make them work with the synchronous base class methods. Let me know if you were thinking about a different approach.

I can definitely change the method names to follow the postfix convention.